packages/react-rest/src/services/react-hooks.ts

```diff
@@ -1,21 +1,83 @@
-import { useState, useEffect } from "react"
```

Comment:
this file (and source-jsonapi) are starting to get a bit large, in my next pr (fetchone) I will break these out to 1 hook/function per file I think, because the test files are getting really big too with each function having 2-3+ tests